packages/batch-submitter/test/batch-submitter/batch-submitter.spec.ts

```diff
@@ -198,6 +198,7 @@ describe('BatchSubmitter', () => {
 
   afterEach(() => {
     sinon.restore()
+    testMetrics.registry.clear()
```

Comment:
I think we should just be mocking the library.

Reply:
didn't find a library around this and the graph's use was much cleaner than ours so didn't need to do this 😭 i've moved this temporarily into the constructor; i will make a note of this in my bs refactor doc, let's observe if this surfaces any problems when we have dashboards running?